lib/node_runner_checker.go

```diff
@@ -320,8 +321,13 @@ func CheckNodeRunnerHandleINITBallotValidateTransactions(c sebakcommon.Checker,
 		return
 	}
 
-	if checker.Ballot.TransactionsLength() < 1 {
-		checker.VotingHole = VotingYES
+	confirmed, _ := time.Parse(sebakcommon.TIMEFORMAT_ISO8601, checker.Ballot.ProposerConfirmed())
```

Comment:
No error checking ?

Reply:
The empty `Transaction`s will not be checked, see #237 .

Reply:
I'm talking about `time.Parse` return value

Reply:
In this state, checking INIT ballot does not check the time is well formatted. It is done by `Ballot.IsWellformed()`.